src/events.js

```diff
@@ -1753,33 +1753,21 @@ export const addGoToCheckInEvent = () => {
 };
 
 export const addGoToSpecimenLinkEvent = () => {
-    // For research
-    const researchSpecimenLinkButtons = document.querySelectorAll('[data-specimen-link-connect-id]');
-    Array.from(researchSpecimenLinkButtons).forEach((btn) => {
-        btn.addEventListener('click', async () => {
+    const workflow = getWorkflow();
```

Comment:
these lines referencing workflow could probably be removed? workflow can only be "research" or "clinical", so doesn't seem like there is a scenario we are trying to keep out with this IF statement

Reply:
Removed.